tests/reporter/kubernetes.go

```diff
@@ -25,11 +25,13 @@ import (
 )
 
 const (
-	sriovEntityURITemplate       = "/apis/sriovnetwork.openshift.io/v1/namespaces/%s/%s/"
-	sriovNetworksEntity          = "sriovnetworks"
-	sriovNodeNetworkPolicyEntity = "sriovnetworknodepolicies"
-	sriovNodeStateEntity         = "sriovnetworknodestates"
-	sriovOperatorConfigsEntity   = "sriovoperatorconfigs"
+	sriovEntityURITemplate            = "/apis/sriovnetwork.openshift.io/v1/namespaces/%s/%s/"
+	sriovNetworksEntity               = "sriovnetworks"
+	sriovNodeNetworkPolicyEntity      = "sriovnetworknodepolicies"
+	sriovNodeStateEntity              = "sriovnetworknodestates"
+	sriovOperatorConfigsEntity        = "sriovoperatorconfigs"
+	k8sCniCncfEntityURITemplate       = "/apis/k8s.cni.cncf.io/v1/namespaces/%s/%s/"
```

Comment:
FYI in golang the common pattern would be `k8sCNICNCFEntityURLTemplate`

Reply:
I changed the name to the right pattern